### DISCUSSION TITLE
Pelco-D

### DISCUSSION THREAD

**Discussion by pedrohbrandao** [glikely/obs-ptz#108] [General]
In this new version do you intend to include the serial control (pelco-d)?

**Comment by pedrohbrandao** [comment]
> In this new version do you intend to include the serial control (pelco-d)?

Because in its new version (obs-ptz v0.12.0-pre2) that has compatibility for obs v28, it doesn't have pelco-d.

**Reply by glikely** [reply]
PELCO-D support will be coming back. There were significant build system changes needed to support v28, but QSerialPort support got broken in the process. I'm working on getting it fixed.

**Reply by pedrohbrandao** [reply]
> O suporte pelco-D vai voltar. Houve mudanças significativas no sistema de construção necessárias para suportar o v28, mas o suporte ao QSerialPort foi quebrado no processo. Estou trabalhando para consertá-lo.

Okay, I really admire your work, I'm a JavaScript programmer, in case you need anything!
